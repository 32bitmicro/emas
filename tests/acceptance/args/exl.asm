exl 0
exl 1
exl 255
