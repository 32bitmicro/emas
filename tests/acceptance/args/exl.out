@ 0x0000 : 0xe500  /  111 001 0 100 000 000  /  58624
@ 0x0001 : 0xe501  /  111 001 0 100 000 001  /  58625
@ 0x0002 : 0xe5ff  /  111 001 0 111 111 111  /  58879
